components/portal/src/img/image.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Is this used anywhere ? If so please update the name to something meaningful. If not please remove

Reply:
Yes, it is the cell image icon. Renamed to CellImage.js